src/main/java/org/gridsuite/config/ConfigService.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.config;
+
+import org.gridsuite.config.dto.ConfigInfos;
+import org.gridsuite.config.repository.ConfigInfosEntity;
+import org.gridsuite.config.repository.ConfigInfosRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.EmitterProcessor;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.function.Supplier;
+import java.util.logging.Level;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+
+@Service
+public class ConfigService {
```

Comment:
Sometimes we use either ConfigUiParameters, configParamaters, ConfigUiInfos, Parameters